dags/hivemind_etl_helpers/src/db/github/extract/__init__.py

```diff
@@ -1,5 +1,6 @@
 # flake8: noqa
 from .comments import fetch_comments
 from .commit import fetch_commits
-from .issues import fetch_issues
+from .github_extraction import GithubExtraction
+from .issues import GithubIssueExtraction
```

Comment:
Please remove line 5 and import the `GithubIssueExtraction` directly from its file in test cases.